sdk/communication/Shared/src/AuthMethod.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
let's keep test code separate from production code.

I'd recommend moving it to `sdk/communication/Shared/tests/`

Reply:
Done